app/locales/es-ES.json

```diff
@@ -181,11 +181,11 @@
   "Some questions and alternatives were not a part of the form from the beginning:": "Algunas preguntas no fueron incluidas desde el inicio:",
   "March 27": "Marzo 27",
   "Other": "Otro",
-  "CALL TO ALL US CITIZENS: This is how we defeat the corona virus: together we make the invisible virus visible.": "LLAMADO A TODOS LOS CIUDADANOS ESPAÑOLES: Así es como derrotamos al virus corona: juntos hacemos visible el virus invisible.",
-  "Contribute and share {{ hostname }} on": "Contribuya y comparta {{ hostname }} en",
-  "Fill out this form - it takes 1 minute - and help the United States gain control over COVID-19.": "Rellene este formulario (se tarda 1 minuto) y ayude a España a obtener el control sobre COVID-19.",
-  "One more thing": "One more thing",
-  "To defeat the corona virus we need to make the invisible virus visible.": "Para vencer al virus corona necesitamos hacer visible el virus invisible.",
+  "CALL TO ALL US CITIZENS: This is how we defeat the corona virus: together we make the invisible virus visible.": "QUERIDOS ESPAÑOLES: Así es como podemos derrotar al coronavirus: juntos podemos hacer visible el virus invisible.",
+  "Contribute and share {{ hostname }} on": "Comparta {{ hostname }} en",
+  "Fill out this form - it takes 1 minute - and help the United States gain control over COVID-19.": "Complete este formulario (se tarda 1 minuto) y ayude a España a controlar el COVID-19.",
+  "One more thing": "Una cosa más",
+  "To defeat the corona virus we need to make the invisible virus visible.": "Para vencer al coronavirus necesitamos hacer visible el virus invisible.",
   "{{ amount }} people already shared their health status.": "{{ amount }} personas ya compartieron su estado de salud.",
-  "Let your friends fill out this form too and help the United States gain control over COVID-19.": "Deje que sus amigos llenen este formulario también y ayude a España a obtener el control sobre COVID-19."
+  "Let your friends fill out this form too and help the United States gain control over COVID-19.": "Dígale a sus amigos que llenen este formulario también y ayude a España a controlar el COVID-19."
```

Comment:
In Spain we would say "rellenar el formulario", not "llenar", so the line would be:

"Let your friends fill out this form too and help the United States gain control over COVID-19.": "Dígale a sus amigos que rellenen este formulario también y ayude a España a controlar el COVID-19."

Not sure in other countries.

Reply:
Hey man, it was me who made the chance for all Spanish versions, sorry if the translation wasn't accurate enough for your country. Please feel free to open a PR to adjust it as it has not been merged yet

Reply:
Fixed this for `app/locales/es-ES.json`